examples/chaincode/go/chaincode_example07/chaincode_example07.go

```diff
@@ -0,0 +1,123 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/hyperledger/fabric/core/chaincode/shim"
+)
+
+// SimpleChaincode example simple Chaincode implementation
+type SimpleChaincode struct {
+}
+
+func (t *SimpleChaincode) Init(stub *shim.ChaincodeStub, function string, args []string) ([]byte, error) {
+	err := stub.PutState("counter", []byte("0"))
+	return nil, err
+}
+
+//Invoke Transaction makes increment counter
+func (t *SimpleChaincode) Invoke(stub *shim.ChaincodeStub, function string, args []string) ([]byte, error) {
+	isOk, _ := stub.VerifyAttribute("position", []byte("Software Engineer"))
+	if isOk {
+		counter, err := stub.GetState("counter")
+		if err != nil {
+			return nil, err
+		}
+		var cInt int
+		cInt, err = strconv.Atoi(string(counter))
+		if err != nil {
+			return nil, err
+		}
+		cInt = cInt + 1
+		counter = []byte(strconv.Itoa(cInt))
+		stub.PutState("counter", counter)
+	}
+	return nil, nil
+
+}
+
+// Deletes an entity from state
+func (t *SimpleChaincode) delete(stub *shim.ChaincodeStub, args []string) ([]byte, error) {
+	if len(args) != 1 {
+		return nil, errors.New("Incorrect number of arguments. Expecting 3")
+	}
+
+	// Delete the key from the state in ledger
+	err := stub.DelState("counter")
+	if err != nil {
+		return nil, errors.New("Failed to delete state")
+	}
+
+	return nil, nil
+}
+
+// Run callback representing the invocation of a chaincode
+// This chaincode will manage two accounts A and B and will transfer X units from A to B upon invoke
+func (t *SimpleChaincode) Run(stub *shim.ChaincodeStub, function string, args []string) ([]byte, error) {
```

Comment:
Why is there a run function? This has been deprecated. It looks like it was just copy/pasted as the comment doesn't seem to match the chaincode